packages/flutter/test/material/flexible_space_bar_collapse_mode_test.dart

```diff
@@ -160,53 +85,12 @@ void main() {
 
     expect(topBeforeScroll.dy, equals(0.0));
     expect(topAfterScroll.dy, equals(-100.0));
-  });
-
-
-
-  testWidgets('FlexibleSpaceBar collapse mode parallax on Android', (WidgetTester tester) async {
-    await tester.pumpWidget(
-      MaterialApp(
-        theme: ThemeData(platform: TargetPlatform.android),
-        home: Scaffold(
-          body: CustomScrollView(
-            key: blockKey,
-            slivers: <Widget>[
-              SliverAppBar(
-                expandedHeight: expandedAppbarHeight,
-                pinned: true,
-                flexibleSpace: FlexibleSpaceBar(
-                  background: Container(
-                    key: appbarContainerKey,
-                  ),
-                  collapseMode: CollapseMode.parallax,
-                ),
-              ),
-              SliverToBoxAdapter(
-                child: Container(
-                  height: 10000.0,
-                ),
-              ),
-            ],
-          ),
-        ),
-      ),
-    );
-
-    final Finder appbarContainer = find.byKey(appbarContainerKey);
-    final Offset topBeforeScroll = tester.getTopLeft(appbarContainer);
-    await slowDrag(tester, blockKey, const Offset(0.0, -100.0));
-    final Offset topAfterScroll = tester.getTopLeft(appbarContainer);
-
-    expect(topBeforeScroll.dy, equals(0.0));
-    expect(topAfterScroll.dy, lessThan(10.0));
-    expect(topAfterScroll.dy, greaterThan(-50.0));
-  });
+  }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.android, TargetPlatform.iOS,  TargetPlatform.macOS }));
```

Comment:
Very cool that this allows you to remove all of the platform specific code above :+1: